run.c

```diff
@@ -339,12 +339,29 @@ human_interaction(double TimeLastPM, double TimeLastOut)
  * be outside the allowed bounds, it will be readjusted by the function ensure_neighbours(), and for those
  * particle, the densities are recomputed accordingly. Finally, the hydrodynamical forces are added.
  */
-void compute_accelerations(int is_PM, int FirstStep)
+void compute_accelerations(int is_PM, int FirstStep, int GasEnabled)
```

Comment:
Mention that the subroutine assumes a valid tree before and after -- Perhaps add an assertion?

Reply:
It is not valid after - starformation may invalidate the tree. Actually we should make this explicit.